lib/stats/series.js

```diff
@@ -0,0 +1,209 @@
+import base from 'taskcluster-base';
+
+const BASE_WORKER_SCHEMA = {
+    workerId: base.stats.types.String,
+    workerType: base.stats.types.String,
+    workerGroup: base.stats.types.String,
+    provisionerId: base.stats.types.String
+};
+
+export const workerStart = new base.stats.Series({
+  name: 'worker.start',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time worker started
+    value: base.stats.types.Number
+  })
+});
+
+export const workerShutdown = new base.stats.Series({
+  name: 'worker.shutdown',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerUptime = new base.stats.Series({
+  name: 'worker.uptime',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+
+export const workerTotalMemory = new base.stats.Series({
+  name: 'worker.memory_total',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerFreeMemory = new base.stats.Series({
+  name: 'worker.memory_free',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerHDUsed = new base.stats.Series({
+  name: 'worker.hd_used',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerHDFree = new base.stats.Series({
+  name: 'worker.hd_free',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerHDTotal = new base.stats.Series({
+  name: 'worker.hd_total',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
```

Comment:
I know that this is what influxdb teaches us to do...
But I'm tempted to merge `workerHDUsed`, `workerHDFree` and `workerHDTotal` and report them:
```js
export const workerHD = new base.stats.Series({
  name: 'worker.hd',
  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
    used: base.stats.types.Number,
    free: base.stats.types.Number,
    total: base.stats.types.Number
  })
});
```
This certain isn't what the 0.9 docs advice, but I wonder if it's wrong??? This way we can ask relative questions like how many have `< 50%` free, etc... Having free space without knowing the size of the disk isn't very fun... Same with `used`. I suspect that don't need both `used` and `free`, but maybe `size`, `used` and `freePerCapacity`.

Also I think you can save this as bytes, `2^53` (max in js), can count to 9 Peta bytes... So counting in GB might also we be worth it, either way comment should reflect this.. btw, I like comments, especially when they aren't duplicates :)